.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+## Pull request checklist
+
+Please check if your PR fulfills the following requirements:
+
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
+- [ ] Build (`npm run build`) was run locally for affected output targets
+
```

Comment:
Should we add an explicit section about running tests? E.g.

```
- [ ] Tests (`npm test`) were run locally and passed
```

There's some overlap there with 'Tests for the changes have been added (for bug fixes / features)', but I'd also argue there's a case where folks won't add tests, but can still run the existing ones to check for regressions